README.md

```diff
@@ -2,32 +2,68 @@ Upstream status (`master` branch):
 [![OfflineIMAP build status on Travis-CI.org](https://travis-ci.org/OfflineIMAP/offlineimap.svg?branch=master)](https://travis-ci.org/OfflineIMAP/offlineimap)
 [![OfflineIMAP code coverage on Codecov.io](https://codecov.io/gh/OfflineIMAP/offlineimap/branch/master/graph/badge.svg)](https://codecov.io/gh/OfflineIMAP/offlineimap)
 [![Gitter chat](https://badges.gitter.im/OfflineIMAP/offlineimap.png)](https://gitter.im/OfflineIMAP/offlineimap)
+[![Say thanks to OfflineIMAP](https://img.shields.io/badge/Say%20Thanks-!-______.svg)](https://saythanks.io/to/OfflineIMAP)
 
 Upstream status (`next` branch):
 [![OfflineIMAP build status on Travis-CI.org](https://travis-ci.org/OfflineIMAP/offlineimap.svg?branch=next)](https://travis-ci.org/OfflineIMAP/offlineimap)
 
-[offlineimap]: http://github.com/OfflineIMAP/offlineimap
-[website]: http://www.offlineimap.org
-[wiki]: http://github.com/OfflineIMAP/offlineimap/wiki
-[blog]: http://www.offlineimap.org/posts.html
+[![Github All Releases](https://img.shields.io/github/downloads/atom/atom/total.svg)](https://github.com/OfflineIMAP/offlineimap/graphs/traffic)
+[![License: GPL v2](https://img.shields.io/badge/License-GPL%20v2-blue.svg)](https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html)
+[![GitHub contributors](https://img.shields.io/github/contributors/OfflineIMAP/offlineimap.svg)](https://github.com/OfflineIMAP/offlineimap/graphs/contributors)
+[![Snap status](https://build.snapcraft.io/badge/snapcrafters/offlineimap.svg)](https://build.snapcraft.io/user/snapcrafters/offlineimap)
 
-Links:
-* Official github code repository: [offlineimap]
-* Website: [website]
-* Wiki: [wiki]
-* Blog: [blog]
+[![Python versions supported](https://img.shields.io/pypi/pyversions/offlineimap.svg)](https://pypi.python.org/pypi/offlineimap)
```

Comment:
Looks wrong. Pypi versions are not python versions. Remove this line.

Reply:
This badge ("pypi python versions supported") is proof that the metadata for python versions supported is missing in pypi.  I have a PR ready to add this metadata so that the next release 7.1.6 will show this metadata in PyPI, as it should.

Reply:
We should have these metadata enabled first, then.

Reply:
Is it possible to edit the contents of the 7.1.5 on pypi and add the metadata into it?  Or, the metadata can be added on this PR for the next release 7.1.6.  But until the 7.1.6 release arrives at pypi, this badge will show "unknown" and "not found" because it queries the record (XML DOAP?) at pypi directly for the metadata of "python version supported" and "release status".

Reply:
Sorry, I have to add the new updated version of `setup.py` for this data to show up.